lib/util/platform.js

```diff
@@ -135,7 +144,8 @@ shaka.util.Platform = class {
    * @return {boolean}
    */
   static isApple() {
-    return !!navigator.vendor && navigator.vendor.includes('Apple');
+    return !!navigator.vendor && navigator.vendor.includes('Apple')
+            && !shaka.util.Platform.isTizen2();
```

Comment:
I don't have data handy about which Tizen versions have "Apple" in navigator.vendor (likely just the ones that are WebKit-based), but I believe it would be safer to just exclude all Tizen versions from isApple().  If it's Tizen, it's not really Apple.